LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License

```

Comment:
I figured we'd go with the MIT License for this project. It's what we're using for our open source FE packages as well.